README.md

```diff
@@ -1,3 +1,43 @@
-# Membership
+# Node.js Foundation - Membership
 
-Repo to work out the system for individual membership in the foundation.
+## What types of members are there?
+
+* Enterprise
+* Individual
+* Student
+* Active Contributor
+
+## What are the conditions of eligibility?
+
+TBD - Join the discussion on this at [issues/8](https://github.com/nodejs/membership/issues/8)
+
+## How to join
+
+To apply to be a member of the Node Foundation, create a new issue in this repository. Use the formatting given below, and preview your post before you submit it to make sure formatting and content are correct.
+
+### Application structure:
```

Comment:
I'm a little uncomfortable with the idea of having an application for membership; provided I'm willing to give the foundation the requested amount of money per year, I should be able to join, quite literally with no questions asked.